tensorflow_addons/metrics/f1_test.py

```diff
@@ -106,6 +110,23 @@ def test_f1_random_score_none(self):
         for avg, res in test_params:
             self._test_f1(avg, actuals, preds, res)
 
+    # Keras model check
+    def keras_check(self):
+        model = tf.keras.Sequential()
+        model.add(layers.Dense(64, activation='relu'))
+        model.add(layers.Dense(64, activation='relu'))
+        model.add(layers.Dense(1, activation='softmax'))
+        fb = F1Score(1, 'macro')
+        model.compile(
+            optimizer='rmsprop',
+            loss='categorical_crossentropy',
+            metrics=['acc', fb])
+        # data preparation
+        data = np.random.random((10, 3))
+        labels = np.random.random((10, 1))
+        labels = np.where(labels > 0.5, 1, 0)
+        model.fit(data, labels, epochs=10, batch_size=32, verbose=0)
```

Comment:
Make `epochs=1`, making it too large will make the test slower, and we just want to check compatibility.